computron/computron.go

```diff
@@ -0,0 +1,438 @@
+package computron
+
+/*
+#cgo  linux CFLAGS: -I${SRCDIR}/python_install/include/python3.11 -I${SRCDIR}/python_install/lib/python3.11/site-packages/numpy/core/include
+#cgo  linux LDFLAGS: -L${SRCDIR}/python_install/lib/combined -lpython3.11-combined -ldl
+
+#cgo  darwin CFLAGS: -I${SRCDIR}/python_install/include/python3.11 -I${SRCDIR}/python_install/lib/python3.11/site-packages/numpy/core/include
+#cgo  darwin LDFLAGS: -L${SRCDIR}/python_install/lib/combined -lpython3.11-combined -ldl
+
+#cgo windows CFLAGS: -I${SRCDIR}/python_install/include/python3.11 -I${SRCDIR}/python_install/lib/python3.11/site-packages/numpy/core/include
```

Comment:
The `CFLAGS` are the same for all of these aren't they? Shouldn't we just vary the `LDFLAGS` then?